vagrant/horizontal-app-scaling/jobs/webapp.nomad

```diff
@@ -17,9 +17,9 @@ job "webapp" {
       policy {
         cooldown = "20s"
 
-        check "avg_instance_sessions" {
+        check "avg_sessions" {
           source = "prometheus"
-          query  = "avg((haproxy_server_current_sessions{backend=\"http_back\"}) and (haproxy_server_up{backend=\"http_back\"} == 1))"
+          query  = "sum(traefik_entrypoint_open_connections{entrypoint=\"webapp\"})/scalar(nomad_nomad_job_summary_running{task_group=\"demo\"})"
```

Comment:
```suggestion
          query  = "sum(traefik_entrypoint_open_connections{entrypoint=\"webapp\"} OR on() vector(0))/scalar(nomad_nomad_job_summary_running{task_group=\"demo\"})"
```

I think we need this to cause the Autoscaler to bring count to 1 before running `hey`. From the tutorial:

> The autoscaler observes that the application is idle and scales it down to the min count of 1.

Without this Prometheus returns no data instead of zero, and the Autoscaler errors with `no metrics available`. Could you check if this fixes the problem for you as well?

Reply:
nice find, accepted!